src/ui/public/visualize/visualize.js

```diff
@@ -47,19 +50,16 @@ define(function (require) {
         $scope.spy.mode = ($scope.uiState) ? $scope.uiState.get('spy.mode', {}) : {};
 
         var applyClassNames = function () {
-          var $spyEl = getSpyEl();
-          var $visEl = getVisEl();
+          var $visEl = getVisContainer();
           var fullSpy = ($scope.spy.mode && ($scope.spy.mode.fill || $scope.fullScreenSpy));
 
-          // external
-          $el.toggleClass('only-visualization', !$scope.spy.mode);
```

Comment:
This was only used for making the spy take up the full container, and was sort of confusing in how it worked. The new method just looks to see if the resulting chart container is < 100px tall.